### ISSUE TITLE
sheets: SourceColumnOffset of 0 is not sending the 0 in API request (PivotGroup)

### ISSUE THREAD

**Issue by Rulox** [googleapis/google-api-go-client#689]
Reference: https://github.com/googleapis/google-api-go-client/blob/master/sheets/v4/sheets-gen.go#L7805

In that line, the `omitEmpty` is making that if I set `SourceColumnOffset` to 0, the request won't include the value `{"sortOrder":"ASCENDING"}` instead of `{"sortOrder":"ASCENDING","sourceColumnOffset":0}` This is causing my code to fail with this 400 error (in a `batchUpdate` call):

```
Error 400: Invalid requests[0].updateCells: At least one pivot group source must be specified., badRequest
```

This code was working before, has anything changed on the API side?:

```
sheetsapi.Request{
		UpdateCells: &sheetsapi.UpdateCellsRequest{
			Start: &sheetsapi.GridCoordinate{
				SheetId:    "ID",
				RowIndex:    0,
				ColumnIndex: 0,
			},
			Rows: []*sheetsapi.RowData{
				&sheetsapi.RowData{
					Values: []*sheetsapi.CellData{
						&sheetsapi.CellData{
							PivotTable: &sheetsapi.PivotTable{
								Source: &sheetsapi.GridRange{
									SheetId:          "ID",
									StartRowIndex:    0,
									StartColumnIndex: 0,
									EndRowIndex:      X,
									EndColumnIndex:   Y,
								},
								Rows: []*sheetsapi.PivotGroup{
									&sheetsapi.PivotGroup{
										SourceColumnOffset: 0,
										ShowTotals:         false,
										SortOrder:          "ASCENDING",
									},
								},
								Columns: []*sheetsapi.PivotGroup{
									// code here
								},
								Values: []*sheetsapi.PivotValue{
									// code here
								},
							},
						},
					},
				},
			},
		},
	}
```

**Comment by codyoss** [comment]
Hey,

To better help you could you please give me a little more information:

- What version of Go are you using?
- What version of google.golang.org/api are you pulling in?
- "This code was working before, has anything changed on the API side?" Did you upgrade a dependency and things stopped working?

`omitempty` is added to each field of all of the req/resp structs. If you need need to send any default values, please use the [ForceSendFields](https://pkg.go.dev/google.golang.org/api/sheets/v4) on the given struct. For example: 

```go
&sheetsapi.PivotGroup{
   SourceColumnOffset: 0,
   ForceSendFields: []string{"SourceColumnOffset"}
}
```

**Comment by Rulox** [comment]
Thanks @codyoss 

Sorry I should have given more info:
* Go version: `go1.13.15 darwin/amd64`
* `google.golang.org/api` version: I've tried with 0.5.0 (the one we have) and I tried with 0.32.0 just in case it was a problem, both failed.

This code runs once in a while, and suddenly it stopped working with this error, so no, no update dependency or anything (it uses its own vendor folder so not a change of a non-vendored dependency being updated either).

Anyways,    `ForceSendFields: []string{"SourceColumnOffset"}` did the trick, thanks for this!
